mne/time_frequency/tests/test_tfr.py

```diff
@@ -131,6 +131,23 @@ def test_time_frequency():
     # computed within the method.
     assert_allclose(epochs_amplitude_2.data**2, epochs_power_picks.data)
 
+    # test that averaging power across tapers when multitaper with
+    # output='complex' gives the same as output='power'
+    epoch_data = epochs.get_data()
+    multitaper_power = tfr_array_multitaper(
+        epoch_data, epochs.info['sfreq'], freqs, n_cycles,
+        output="power")
+    multitaper_complex = tfr_array_multitaper(
+        epoch_data, epochs.info['sfreq'], freqs, n_cycles,
+        output="complex")
+
+    print(multitaper_complex.shape)
+    print(multitaper_power.shape)
```

Comment:
Cruft? But probably okay since `pytest` will capture it anyway

Reply:
haha, yes :) I will remove it.